mods/storage/src/storage.ts

```diff
@@ -200,7 +236,29 @@ export default class Storage extends FonosService {
   }
 
   // Internal API
-  getObjectURLSync(request: GetObjectURLRequest) {
+  /**
+   * Get Object URL.
```

Comment:
Consider enhancing comment to clarify that this is a synchronous version of another function.